system/nagios-status-check/detectors-nagios-status-check.tf

```
@@ -0,0 +1,27 @@
+resource "signalfx_detector" "status_check" {
+  name = "${join("", formatlist("[%s]", var.prefixes))}[${var.environment}] Nagios check status"
+
+  program_text = <<-EOF
+        signal = data('nagios_state.state', filter=${module.filter-tags.filter_custom})${var.status_check_aggregation_function}.${var.status_check_transformation_function}(over='${var.status_check_transformation_window}').publish('signal')
```

I udpated other modules code to merge transformation function into only one variable like for the aggregation

```suggestion
        signal = data('nagios_state.state', filter=${module.filter-tags.filter_custom})${var.status_check_aggregation_function}${var.status_check_transformation_function}.publish('signal')
```